docs/agents/sematext-agent/agent-troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+title: Agent Troubleshooting
+
+If you are experiencing issues with the Sematext Agent, there are some things you can try.
+
+1. Check the Sematext Agent logs and filter by errors. By default they'll be located at `/opt/spm/spm-monitor/logs/st-agent`. 
+   1. If you see errors related to permission denied while accessing log files or system information, stop the agent and start it again with elevated privileges. 
```

Comment:
How do I stop/start the agent?  Link to docs needed?

Reply:
Added links to both restarting the agent and the permissions needed.